crates/control/src/role_assignment.rs

```diff
@@ -642,6 +663,10 @@ fn pick_role_with_penalties(
         four: None,
         five: None,
     };
+
+    // assign essential roles: Keeper xor ReplacementKeeper, and Striker
+
+    // Striker
```

Comment:
Remove these comments and replace them by properly named function calls.

Reply:
done